UI/src/view/point_asset/massTransitMarker.js

```diff
@@ -10,6 +10,15 @@
     var EMPTY_IMAGE_TYPE = '99';
     var styleScale = 1;
 
+    var SERVICE_POINT_IMAGES = [
+        {value: 5, imgUrl: 'images/service_points/railwayStation2.png'},
+        {value: 6, imgUrl: 'images/service_points/railwayStation.png'},
+        {value: 7, imgUrl: 'images/service_points/subwayStation.png'},
+        {value: 8, imgUrl: 'images/service_points/airport.png' },
+        {value: 9, imgUrl: 'images/service_points/ferry.png' },
+        {value: undefined, imgUrl: 'images/service_points/' }
```

Comment:
Why we have a element with value "undefined"? The imgUrl will not fetch the png since with that path it does not "exist".

Reply:
Removed